### PR TITLE
Component alerts

### PR DESCRIPTION
**Pull request by sah-shah7**
-Added Alerts Component
image

### CODE REVIEW

File: Components/Alerts/alert.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  position: relative;
+  border-left: 0.5rem solid #ffd48a;
+  color: #856404;
+}
+
+.alert-success {
+  position: relative;
+  color: #155724;
+  background-color: #d4edda;
+  border-color: #c3e6cb;
```
Comment (DevNirah7) [inline Components/Alerts/alert.css:46]: Can use variables for colours

**Review by DevNirah7 — APPROVED** [review]
1. Can use color variables wherever possible.

Rest all looks good to commit!

File: Components/Alerts/alert.css (hunk 1 above)
Comment (sah-shah7) [reply Components/Alerts/alert.css:46]: Hey, thanks. Will add a variable css sheet soon

File: Components/Alerts/alert.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
+
+.alert {
+  box-sizing: border-box;
+  width: 100%;
+  border-radius: 5px;
+  padding: 1.25rem 2.5rem;
+  box-shadow: 0 0 15px 5px #ccc;
+  border-left: 0.5rem solid;
```
Comment (AtulHande03a) [inline Components/Alerts/alert.css:8]: using rem and px , and instead of 0.5rem , it could have been in px??

File: Components/Alerts/alert.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  cursor: pointer;
+}
+
+.close-btn .fa-times {
+  font-size: 1.375rem;
+  line-height: 2.5rem;
+}
+
+.alert-warning {
+  background-color: #fff3cd;
```
Comment (AtulHande03a) [inline Components/Alerts/alert.css:36]: create variables for the colors and then use it

File: Components/Alerts/alerts.html [hunk 4]
```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```
Comment (AtulHande03a) [inline Components/Alerts/alerts.html:6]: add author meta tag

File: Components/Alerts/alerts.html [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+                    <li class="sidebar-list-items">
+                        <a href="../../Documentation/documentation.html" class="link-a link-a-font-color-black sidebar-list-items-text">Getting
+                            Started</a>
+                    </li>
+
+                    <li class="sidebar-list-items">
+                        <a href="../../Components/Alerts/alerts.html" class="link-a link-a-font-color-black sidebar-list-items-text">Alerts</a>
+                    </li>
+                    <li class="sidebar-list-items">
+                        <a href="../../Components/" class="link-a link-a-font-color-black sidebar-list-items-text">Avatars</a>
```
Comment (AtulHande03a) [inline Components/Alerts/alerts.html:54]: the naming is very big "link-a-font-color-black" .. stick to link-font or link-font-color like this for example

**Review by AtulHande03a — CHANGES_REQUESTED** [review]
color variables, long class names , px vs rem units usage .. apart from this rest looks okay

File: Components/Alerts/alert.css (hunk 2 above)
Comment (sah-shah7) [reply Components/Alerts/alert.css:8]: Thanks for bringing it to my attention. Making the changes

**Pull request merged**